recipes/great-expectations/00-force-long_description.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/setup.py b/setup.py
+index 6877517..18df129 100644
+--- a/setup.py
++++ b/setup.py
+@@ -7,8 +7,7 @@ except ImportError:
+ with open('requirements.txt') as f:
+     required = f.read().splitlines()
+ 
+-with open('docs/source/intro.rst') as f:
+-    long_description = f.read()
++long_description = ''
```

Comment:
Why is this needed?

Reply:
The upstream code is not shipping the "docs" folder so setup.py fails with that line in there.

This is is fixed in their master branch and will be removed in the next release.